src/main/java/com/ibm/disni/verbs/impl/RdmaVerbsNat.java

```diff
@@ -217,4 +217,27 @@ public int destroyCQ(IbvCQ cq) throws IOException {
 		int ret = nativeDispatcher._destroyCQ(cqImpl.getObjId());
 		return ret;
 	}
+
+	@Override
+	public void checkResources(IbvContext context, int maxWR, int maxSge, int cqSize) throws IOException {
```

Comment:
Or even change this method to `long queryDevice` that would return a pointer to `ibv_device_attr` structure, then we can remove `queryOdpSupport` also and check other device capabilities.